tools/cal_scripts/run_all_parsers.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import os, glob, subprocess
+import time
+
+base_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../..'))
+
+def main():
+    for sub in os.listdir(os.getcwd()):
+        if os.path.isdir(sub):
+            os.chdir(sub)
+            # Checks if there are any cal files in Manufacturer Cal Files and do not
+            # redo sheets that have already been made.
+            if len(os.listdir('manufacturer')) != 0:
+                for file in glob.glob('*.py'):
```

Comment:
`glob('*.py')` will match run_all_parsers.py, this will result in this script calling itself. How about `glob('*cal_parser.py')`?